src/Kiosk.Abstractions/Models/Events/Event.cs

```diff
@@ -0,0 +1,25 @@
+
+
+using MongoDB.Bson;
+using MongoDB.Bson.Serialization.Attributes;
+
+namespace Kiosk.Abstractions.Models.Events;
+
+public class EventDetails
+{
+    public required string name { get; set; }
+    public required string content { get; set; }
+}
+
+public class Event
+{
+    [BsonId]
+    [BsonRepresentation(BsonType.ObjectId)]
+    
+    public string? _id { get; set; }
+    
+    public required string url { get; set; }
```

Comment:
url property is not required, since admin may want to add his own custom event. Also make sure to set property names starting with capital letter to follow c# best code formatting practices